install/installer/pkg/components/iam/objects.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+package iam
+
+import (
+	"github.com/gitpod-io/gitpod/installer/pkg/common"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func Objects(ctx *common.RenderContext) ([]runtime.Object, error) {
+	return common.CompositeRenderFunc(
```

Comment:
This will be default deploy it to all installations, including self-hsoted. I think this is okay, just highlighting it here.